smqtk_classifier/impls/classification_element/postgres.py

```diff
@@ -237,10 +222,10 @@ def get_classification(self) -> CLASSIFICATION_DICT_T:
             "uuid_val": str(self.uuid)
         }
 
-        conn = self._get_psql_connection()
+        conn = self._psql_helper.get_psql_connection()
```

Comment:
Here we want to replace most of body with the use of the `self._psql_helper.single_execute` method. See the `get_bytes` method in `smqtk_dataprovider/impls/data_element/psql.py` for example.